src/aspire/source/image.py

```diff
@@ -159,6 +160,8 @@ def __init__(self, L, n, dtype="double", metadata=None, memory=None):
 
         # Instantiate the accessor for the `images` property
         self._img_accessor = _ImageAccessor(self._images, self.n)
+        # For base ImageSource, signal is estimated from `images`
+        self._signal_images = self.images
```

Comment:
That is true, but his is just for the base ImageSource, where we have no "clean" images. Those (clean) are only available in `Simulation`, where this is overloaded (unless I made a mistake).

I think this is consistent with (some of) the MATLAB scripts , at least to the extent of trying to measure signal and center, and noise as outer, when dealing with unknown images.

What would you rather see that we already have implemented in the case of noisy images?

Reply:
This has been refactored following the meeting.

The base Image source now provides the same estimate_signal_power methods for all classes, which by default use self.images (ie, includes noise), and returns `snr = (signal_power - noise_power) / noise_power`

Reply:
Simulation additionally provides a `true_signal_power` that instead uses `clean_images`.  

This approach must be used for the `from_snr` calls anyway, since we can't measure noise which isn't generated yet...

Reply:
Thanks! :+1: